livekit-server/.env.example

```
@@ -0,0 +1,7 @@
+LIVEKIT_API_KEY=
+LIVEKIT_API_SECRET=
+PORT=8080
+LIVEKIT_NODE_IP_MODE=proxy
+LIVEKIT_LOG_LEVEL=info
+LIVEKIT_MONITOR_WEBHOOK_URL=
+REDIS_URL=${{Redis.REDIS_URL}}
```

dotenv-linter (4.0.0)

[warning] [SubstitutionKey] The REDIS_URL key is not assigned properly

(SubstitutionKey)